_sass/components/_card.scss

```diff
@@ -0,0 +1,37 @@
+.card {
```

Comment:
Minor: There's already a [login.gov Design System Card](https://design.login.gov/components/cards/) and [USWDS Card](https://designsystem.digital.gov/components/card/). Is there a more specific name we could use, or use one of those options as given if it fits the purpose? Or make it something more specific to the help section and less of a site-wide "card" component?

Reply:
This is the one and same LDS card! The goal is to make it as flexible as the USWDS card in terms of media positioning. It will be used sitewide for list or feed content that requires: an image, title, description, call to action.

Reply:
See my very WIP doc on [proposed content structures](https://docs.google.com/document/d/1DOrZBhqg5Jojm-EiyGJVj0ILVgQDZxJ2B6JKQgq7NTw/edit#heading=h.ezbrrmdm48my).